desktop/src/main/java/bisq/desktop/primary/main/left/LeftNavView.java

```diff
@@ -336,18 +347,13 @@ private void updateSubmenu() {
         LeftNavButton selectedLeftNavButton = model.getSelectedNavigationButton().get();
         for (int i = 0; i < mainMenuItems.getChildren().size(); i++) {
             Node item = mainMenuItems.getChildren().get(i);
-            if (item instanceof VBox) {
-                VBox submenu = (VBox) item;
+            if (item instanceof VBox submenu) {
```

Comment:
As stated in the other PRs commont we want to stick with Java 11 featureset for Android support.